src/highlight.js

```diff
@@ -967,8 +964,7 @@ const HLJS = function(hljs) {
   @returns {string}
   */
   function deprecateFixMarkup(arg) {
-    logger.deprecated("10.2.0", "fixMarkup will be removed entirely in v11.0");
-    logger.deprecated("10.2.0", "Please see https://github.com/highlightjs/highlight.js/issues/2534");
+    logger.deprecated("10.2.0", "fixMarkup will be removed entirely in v11.0. Please see https://github.com/highlightjs/highlight.js/issues/2534");
```

Comment:
Please don't change the invokes, this is going to conflict with other work.  The only thing we should need to touch here if `logger.js`. Thanks.

Reply:
Yeah no need for that if no need for depd.